avalon/tools/contextmanager/app.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, parent=None):
         task_view.setIndentation(0)
         task_model = TasksModel()
         task_view.setModel(task_model)
+        task_view_selection = task_view.selectionModel()
```

Comment:
By keeping the `QItemSelectionModel` object before connecting signal, avoids the `Segmentation fault` crash in Houdini which described above.